repository/src/main/resources/db/migration/V20210107__RDM-10588_dynamic_radio_multi_select_list.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO public.field_type (created_at, reference, version)
```

Comment:
for the future remember the new naming convention we agreed for Flyway files:
`V20210107_10588__RDM-10588_dynamic_radio_multi_select_list.sql `